package.json

```diff
@@ -92,5 +92,8 @@
   },
   "files": [
     "dist"
-  ]
+  ],
+  "dependencies": {
+    "@mantine/dates": "^6.0.18"
+  }
```

Comment:
I've been using devDeps + peerDeps to make sure that it's not inlined and instead required by user. You can check bundler analyzer to see how it's being bundled

https://github.com/developit/microbundle/wiki/How-Microbundle-decides-which-dependencies-to-bundle#how-microbundle-decides-inline-vs-external

Reply:
Make sure when adding to peer deps you keep the min version the same as what's in there

Reply:
Sorry, it was a busy week, didn't get a chance to get back on it.

Ops. Right, I updated `package.json`. Let me know if it looks good to you.